recipes/meta/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "meta" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "a35dd5bc3ccffaaf83aa488233467ae85dc5374aea3068c34a6dc85ea29e27a9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bagari/meta/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('meta', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make  # [not win]
+
+  host:
+    - python
+    - pip
+    - boost
+    - itk >=5.0.0,<=5.3.0
```

Comment:
The problem with the itk package on conda-forge is that it was never build correctly....
https://github.com/conda-forge/itk-feedstock/issues/29

I'm very apprehensive to personally merge anything that depends on it until the issue from 2022 is resolved.

It is a hard one, i'm sorry you are hitting this as a new contributor, but the ITK python package status should be resolved in my opinion before moving on.